gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.3.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
Did you run the new wrapper call? I'm thought more changed than just the ZIP between 5.3 and 5.4.

Reply:
I thought so as well and just re-ran, but no new changes.

```bash
./gradlew wrapper --gradle-version 5.4.1 --distribution-type all --no-daemon --stacktrace
```

See anything that looks off?